### PR TITLE
Initiated system management support

### PR DESCRIPTION
**Pull request by tcharding**
This patch adds systemd support for running OpenBazaar as a daemon. Has bug that config file is not found (because systemd does not run ob from within the repo directory). This bug is unrelated to systemd support however.

### CODE REVIEW

**Comment by cpacia** [comment]
I'm not that familiar here if someone else wants to take a look at this. Also @tcharding I don't think you want your home folder hard coded in there.

**Comment by tyler-smith** [comment]
I think I like the concept. I think many people will still want/need custom configurations for these things, but we could perhaps provide baselines that works for most people.

Would the idea be to provide scripts/support for a bunch of common systems, or just one? A lot of people still prefer to use upstart. I use personally am using Docker on CoreOS. How do we decide which things to include support for?

I think the main issue I see is that to run OB well requires quite a bit more setup/configuration than I think we might be able to reasonably provide here. E.g. it shouldn't run as root, files should have tight permissions, etc.

TL;DR Overall I like the idea, but think we should think more about how we want to execute on it.
